src/components/SessionList/SessionList.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import style from './SessionList.module.scss';
+import SessionListItem from '../SessionListItem/SessionListItem';
+
+const SessionList = ({title, sessions}) => {
+    return(
+        <div className = {style.wrapper}>
+			<div className= {style.heading2}>
```

Comment:
The code should have different spacing, make sure to only use one tab when nesting divs